src/API/IResultsService.cs

```diff
@@ -1,25 +1,25 @@
 ﻿/*
  * Apache License, Version 2.0
  * Copyright 2019-2020 NVIDIA Corporation
- * 
+ *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
  *     http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
 
+using Nvidia.Clara.ResultsService.Api;
```

Comment:
Actually, shouldn't the app assembly be lower in the list than the system ones?

Reply:
I'm using CodeMaid plug-in for code clenaup and here explains why https://stackoverflow.com/questions/29013521/codemaid-vs-stylecop-usings-organisation